third_party/txt/src/txt/platform_mac.mm

```diff
@@ -43,39 +53,50 @@
 }
 
 CTFontRef MatchSystemUIFont(float desired_weight, float size) {
```

Comment:
The ownership of the CTFontRef isn't clear. Perhaps return fml::CFRef<CTFontRef> instead.

Reply:
done.